### PR TITLE
adding info on Personal Access Token and other misc things

### PR DESCRIPTION
**Pull request by dennisseah**
1. adding info on Personal Access Token. This token is needed to create variable group
2. include the need of -d parameter in `create-variable-group` command.

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -138,7 +142,7 @@ application repositories
   was not intialized)
   ```
   VARIABLE_GROUP_NAME=<my-vg-name>
-  spk project create-variable-group $VARIABLE_GROUP_NAME -r $ACR_NAME -u $SP_APP_ID -t $SP_TENANT -p $SP_PASS
+  spk project create-variable-group $VARIABLE_GROUP_NAME -d $HLD_REPO_URL -r $ACR_NAME -u $SP_APP_ID -t $SP_TENANT -p $SP_PASS
```
Comment (mtarng) [inline docs/getting-started.md]: @dennisseah is `-d $HLD_REPO_URL` required? It should read from the config for the value.

**Review by sarath-p — APPROVED** [review]
Looks good

File: docs/getting-started.md (hunk 1 above)
Comment (sarath-p) [reply docs/getting-started.md]: @mtarng - you are correct. It is an optional parameter and falls back to SPK config when it is not specified.

Comment (dennisseah) [reply docs/getting-started.md]: > @dennisseah is `-d $HLD_REPO_URL` required? It should read from the config for the value.

i got an error if I do not provide it.

Comment (dennisseah) [reply docs/getting-started.md]: @sarath-p is able to run it without -d option. I will amend this PR.

**Review by mtarng — APPROVED** [review]

**Pull request merged**